### ISSUE TITLE
Remember save directory separate from export directory.

### ISSUE THREAD

**Issue by garretwilson** [umlet/umlet#450]
I keep my UXF files in one directory, separate from the directory in which I export to e.g. SVG. (Keeping source and output files separate is a common practice in software development.) UMLet standalone seems to remember only one last directory used, conflating the two concepts.

Thus if I load a file in `FolderA`, when it comes time to export the file, the directory listed is FolderA, and I have to go browsing through my hard drive to find `FolderB`. But once I export the file to `FolderB`, when I want to load a new file (or save a new file), UMLet puts me in `FolderB` and I have to go browsing through the entire hard drive again to find `FolderB`.

This is irritating, especially since a natural workflow is to 1) create a file, 2) export it, and then repeat --- UMLet has be browsing and rebrowsing at every step.

Please remember the last-save-directory separately from the last-export-directory.

**Comment by gerritgriebel** [comment]
Good idea! But "natural workflow is to 1) create a file, 2) export it, and then repeat" could be much easier if options would allow additionally to specify automatic export on save. Either a single drop down for desired format (single export) or checkboxes (multiple exports).

**Comment by MarqueIV** [comment]
+1 to both of the above.
